### PR TITLE
Added a Dockerfile

### PR DESCRIPTION
**Pull request by intellitour**
To help those willing to use this *pub_server* as is.

### CODE REVIEW

**Comment by intellitour** [comment]
@googlebot I signed it!

**Comment by thotty** [comment]
Great, @intellitour
 
This will help a lot!

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+RUN mkdir /app && cd /app && git clone https://github.com/dart-lang/pub_server.git
+RUN cd /app && wget https://storage.googleapis.com/dart-archive/channels/stable/release/latest/linux_packages/dart_2.7.1-1_amd64.deb && dpkg -i dart_2.7.1-1_amd64.deb
+RUN PATH="$PATH:/usr/lib/dart/bin" && chmod -R +x /usr/lib/dart/bin/*
+
+RUN adduser pubserver --system -q && chown -R pubserver /app
+USER pubserver
+
+WORKDIR /app/pub_server
+RUN cd /app/pub_server && /usr/lib/dart/bin/pub get
+VOLUME [ "/app/packages-db" ]
```
Comment (jonasfj) [inline Dockerfile:15]: I don't recall flutter defaults anymore, but I think this defaults to leaving a volumen on the system.. Might be fine, but seems like something that should be documented

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:latest
```
Comment (jonasfj) [inline Dockerfile:1]: We have Dart docker images here: https://hub.docker.com/r/google/dart-runtime/

**Review by jonasfj — COMMENTED** [review]
I honestly, think it's better to contribute to one of the community projects like: https://pub.dev/packages/unpub

@mit-mit, maybe we should update the README, link to community projects and declare this repository/package for discontinued.

We don't really have cycles to review changes in this repository. And changes beyond adding a harmless `Dockerfile` are hard to do because we use this package on `pub.dev`. So I think energy towards building a self-hosted pub server is better spent in other projects.

**Comment by mit-mit** [comment]
Closing this PR: This repo is not actively maintained (readme will be updated shortly)

**Pull request closed without merging**